chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/CertifiedProductDetailsManagerImpl.java

```diff
@@ -1,599 +1,645 @@
+import gov.healthit.chpl.dto.CertificationResultTestStandardDTO;
+import gov.healthit.chpl.dto.CertificationResultTestTaskDTO;
+import gov.healthit.chpl.dto.CertificationResultTestToolDTO;
+import gov.healthit.chpl.dto.CertificationResultUcdProcessDTO;
+import gov.healthit.chpl.dto.CertificationStatusDTO;
+import gov.healthit.chpl.dto.CertificationStatusEventDTO;
+import gov.healthit.chpl.dto.CertifiedProductAccessibilityStandardDTO;
+import gov.healthit.chpl.dto.CertifiedProductDetailsDTO;
+import gov.healthit.chpl.dto.CertifiedProductQmsStandardDTO;
+import gov.healthit.chpl.dto.CertifiedProductTargetedUserDTO;
+import gov.healthit.chpl.dto.MacraMeasureDTO;
+import gov.healthit.chpl.dto.TestParticipantDTO;
+import gov.healthit.chpl.manager.CertificationResultManager;
+import gov.healthit.chpl.manager.CertifiedProductDetailsManager;
+import gov.healthit.chpl.manager.SurveillanceManager;
+import gov.healthit.chpl.util.CertificationResultRules;
+
+@Service("certifiedProductDetailsManager")
+public class CertifiedProductDetailsManagerImpl implements CertifiedProductDetailsManager {
+	private static final Logger logger = LogManager.getLogger(CertifiedProductDetailsManagerImpl.class);
+
+	@Autowired
+	private CertifiedProductSearchResultDAO certifiedProductSearchResultDAO;
+	
+	@Autowired
+	private CQMResultDetailsDAO cqmResultDetailsDAO;
+	
+	@Autowired private CQMResultDAO cqmResultDao;
+	
+	@Autowired
+	private CertificationResultDetailsDAO certificationResultDetailsDAO;
+	
+	@Autowired
+	private CertifiedProductQmsStandardDAO certifiedProductQmsStandardDao;
+	
+	@Autowired CertifiedProductTargetedUserDAO certifiedProductTargetedUserDao;
+	@Autowired CertifiedProductAccessibilityStandardDAO certifiedProductAsDao;
+	
+	@Autowired
+	private CertificationResultManager certResultManager;
+	
+	@Autowired
+	private CertificationStatusEventDAO certStatusEventDao;
+	
+	@Autowired
+	private CertificationStatusDAO certStatusDao;
+	
+	@Autowired
+	private CertificationResultRules certRules;
+	
+	@Autowired private ListingGraphDAO listingGraphDao;
+
+	@Autowired private SurveillanceManager survManager;
+	
+	private CQMCriterionDAO cqmCriterionDAO;
+	private MacraMeasureDAO macraDao;
+	
+	private List<CQMCriterion> cqmCriteria = new ArrayList<CQMCriterion>();
+	private List<MacraMeasure> macraMeasures = new ArrayList<MacraMeasure>();
+	
+	@Autowired
+	public CertifiedProductDetailsManagerImpl(CQMCriterionDAO cqmCriterionDAO, MacraMeasureDAO macraDao){
+		this.cqmCriterionDAO = cqmCriterionDAO;
+		this.macraDao = macraDao;
+		
+		loadCQMCriteria();
+		loadCriteriaMacraMeasures();
+	}
+	
+	@Override
+	@Transactional
+	public ArrayList<IcsFamilyTreeNode> getIcsFamilyTree(Long certifiedProductId) throws EntityRetrievalException {
+		
+		ArrayList<IcsFamilyTreeNode> familyTree = new ArrayList<IcsFamilyTreeNode>();
+		Map<Long,Boolean> queue = new HashMap<Long,Boolean>();
+		ArrayList<Long> toAdd = new ArrayList<Long>();
+		
```

Comment:
Similar comment as above - the <Map> is consistent with the CHPL coding style but ArrayList<> lines are not. I would expect something like 
List<IcsFamilyTreeNode> familyTree = new ArrayList<IcsFamilyTreeNode>();